tests/jobs/replace_string/test_integration.py

```diff
@@ -8,16 +7,21 @@
 QUEUE_SYSTEM LOCAL
 NUM_REALIZATIONS 1
 FORWARD_MODEL COPY_FILE(<FROM>=<CONFIG_PATH>/file.txt, <TO>=file.txt)
-FORWARD_MODEL REPLACE_STRING(<FROM>={FROM}, <TO>={TO}, <FILE>=file.txt)
+FORWARD_MODEL REPLACE_STRING(<FROM>={FROM}, <TO>="{TO}", <FILE>=file.txt)
```

Comment:
There is only quotes on `"{TO}"`, I am sure there is a reason for that, and that reason might be very well suited for the documentation for this job?

Reply:
This is due to the general problem with ERT and how it escapes strings and how it is parsing arguments for forward model jobs.

Reply:
Would be great to have that in the documentation as it is a common question.

Reply:
done!